examples/siro_sandbox/sandbox_app.py

```diff
@@ -821,6 +875,14 @@ def _update_help_text(self):
                     text_delta_x=-150,
                     text_delta_y=-50,
                 )
+
+            progress_str = f"{self._num_iter_episodes - (self._num_episodes_done + 1)} episodes remaining"
```

Comment:
We have some help text in `_get_status_text` that instructs the user to press M at the end of the episode. Can you update this, as well? Help them understand when they're done and can just quit the application.

Reply:
I decided to remove `Press M to try again on the next episode.` from the status string since when task is complete or something went wrong, control string changes and contains only "ESC" and (when next episode exists) "M". This way our control and status strings are decoupled nicely and don'd contain duplicate information.